java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/protocol/TextDocumentServiceImpl.java

```diff
@@ -235,7 +236,6 @@
     /**
      * Documents actually opened by the client.
      */
-    private final Map<String, Document> openedDocuments = new ConcurrentHashMap<>();
```

Comment:
`OpenDocuments` class is a refactored version of the map then, right?

Reply:
Yes, it's a wrapped map with added ability to listen for changes (consume the added content).